test/e2e/specs/authentication-spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Authentication', () => {
+  beforeEach(() => {
+    cy.task('createUser', {
+      email: 'masa',
```

Comment:
should these email and username be extracted into variables within the describe context?

Reply:
Done :+1: